pythonforandroid/bootstraps/service_only/build/src/org/kivy/android/PythonService.java

```diff
@@ -136,7 +131,6 @@ public void onDestroy() {
     @Override
     public void run() {
         PythonUtil.loadLibraries(getFilesDir());
-        mService = this;
```

Comment:
I'm concerned that some users may be using a single service and relying on this method of accessing it, as that's the way the old toolchain worked. Would you be terribly disappointed if I merged this but added mService back in for now (at least until after it's clearly deprecated)?